core-chart/templates/_helpers.tpl

```diff
@@ -1,6 +1,10 @@
 {{/*
 Expand ITS PCH custeradm container.
 */}}
+
+{{/*
+ITS clusteradmin container for PostCreateHook

```

Comment:
"ITS clusteradmin" is not a very meaningful phrase. A more informative comment would explain that this container initializes the cluster as an OCM "hub".